codespell_lib/data/dictionary.txt

```diff
@@ -1624,6 +1645,7 @@ atheneans->athenians
 athiesm->atheism
 athiest->atheist
 athough->although
+athron->athlon
```

Comment:
It this is the brand, should it be `Athlon`

See https://en.wikipedia.org/wiki/Athlon

Reply:
I guess so, also.

Thanks for the review.

Reply:
fixed